dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageError.java

```diff
@@ -44,6 +44,8 @@
   /** Returns the reason why the symbol reference is marked as a linkage error. */
   abstract Reason getReason();
 
+  abstract boolean isSourceClassReachable();
```

Comment:
I think `isReachable` is enough here. No need for SourceClass. 
This does need JavaDoc.